pages/mentor/22-student_-_mentor_facilitation.md

```diff
@@ -2,7 +2,7 @@
 title: GSoC contributor - Mentor Facilitation
 section: Org Admin
 order: 22
-permalink: mentor/GSoC contributor-mentor-facilitation.html
+permalink: mentor/contributor-mentor-facilitation.html
```

Comment:
Changing the permalink will also break the existing URL.

Reply:
So you'd like this line to be:

```
url: GSoC%20contributor-mentor-facilitation
```

And have the production URL remain `https://google.github.io/gsocguides/mentor/GSoC%20contributor-mentor-facilitation`?

Reply:
Is it possible to add a redirect to the configuration so that the URL doesn't have to remain that permanently?

Reply:
No, I think it should be https://google.github.io/gsocguides/student-mentor-facilitation.  The GSOC%20 bit got missed in review.

From a very small amount of digging, I don't believe redirects are simple.  

It might be possible to create a page with a `<meta http-equiv="refresh" content="0;URL='http://blah.example.com/'" />` that does the right thing.  It would have to be excluded from any auto-indexes, etc.

Reply:
@rspier, ok, I'm still a little unclear on what the change you're looking for is here, but I've force-pushed another attempt. 🙂 If that's still not your ideal fix, feel free to close this PR and make a different one with that fix, I unfortunately can't spend any more cycles on this, hope you understand.